docs/xml-sitemaps.md

```diff
@@ -2,70 +2,166 @@
-
-## Adding an XML namespace
+The provider will be responsible for getting all sitemaps and sitemap entries, as well as determining pagination.
+
+## Removing Specific Sitemaps
+There are three existing sitemaps providers for standard object types - `posts`, `taxonomies`, and `users`. If you want to remove one of them such as the "users" provider, you can configure the module like so:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"seo": {
+					"xml-sitemaps": {
+						"users": false
+					}
+				}
+			}
+		}
+	}
+}
+```
 
-Some custom extensions to the standard sitemap format are supported but require the addition of one or more XML namespaces for them to validate. This is achieved using the `msm_sitemap_namespace` filter. The below example adds Google's video sitemap schema.
+Alternatively if you need to use more complex logic or work with custom sitemap providers you can use the `wp_sitemaps_add_provider` filter to do so for example:
 
 ```php
-add_filter( 'msm_sitemap_namespace', function ( array $namespaces ) : array {
-	$namespaces['xmlns:video'] = "http://www.google.com/schemas/sitemap-video/1.1";
-	return $namespaces;
-} );
+add_filter( 'wp_sitemaps_add_provider', function ( $provider, string $name ) {
+	// Only switch off the user sitemaps for subsites on the network.
+	if ( $name === 'users' && ! is_main_site() ) {
+		return false;
+	}
+
+	return $provider;
+}, 10, 2 );
 ```
 
-## Skipping a post
+If instead you want to disable sitemap generation for a specific post type or taxonomy, use the `wp_sitemaps_post_types` or `wp_sitemaps_taxonomies` filter, respectively.
 
-The `msm_sitemap_skip_post` filter is used to exclude posts from the sitemaps programmatically. The current post is available via `get_post()`.
+To disable sitemaps for the page post type you would do the following:
 
 ```php
-add_filter( 'msm_sitemap_skip_post', function () {
-	if ( get_post_type() === 'hideme' ) {
-		return true;
-	}
+add_filter( 'wp_sitemaps_post_types', function ( array $post_types ) : array {
+	unset( $post_types['page'] );
+	return $post_types;
+} );
+```
+
+To disable sitemaps for the `post_tag` taxonomy:
 
-	return false;
+```php
+add_filter( 'wp_sitemaps_taxonomies', function ( array $taxonomies ) : array {
+	unset( $taxonomies['post_tag'] );
+	return $taxonomies;
 } );
 ```
 
-## CLI commands
+## Adding Additional Tags to Sitemap Entries
+The sitemaps protocol specifies a certain set of supported attributes for sitemap entries. Of those, only the URL (loc) tag is required. All others (e.g. `changefreq` and `priority`) are optional tags in the sitemaps protocol and not typically consumed by search engines, which is why only the URL itself is listed by default. You can still add those tags if you need to.
```

Comment:
What is "the (loc) tag"? Is it code? If so, it should be in backticks. Does it need to be wrapped in parentheses? (`changefreq` and `priority` aren't)

Reply:
Yeah good shout, fwiw I mostly copy pasted this from the Make blog post 😄 

```suggestion
The sitemaps protocol specifies a certain set of supported attributes for sitemap entries. Of those, only the `loc` tag (URL) is required. All others (e.g. `changefreq` and `priority`) are optional tags in the sitemaps protocol and not typically consumed by search engines, which is why only the URL itself is listed by default. You can still add those tags if you need to.
```